helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -243,6 +243,18 @@ public Response updateCluster(@PathParam("clusterId") String clusterId,
         }
         break;
 
+      case deactivate:
+        if (superCluster == null) {
```

Comment:
can empty string be possible?

Reply:
I followed the same pattern as `activate`, but just noticed there is a cluster check for activate, but not for deactivate. Will add that check. Thanks.